langkit/templates/pkg_init_body_ada.mako

```diff
@@ -2,10 +2,53 @@
 
 with GNATCOLL.Iconv;
 
+% if emitter.coverage:
+with Ada.Environment_Variables;
+
+with Interfaces.C;
+
+with GNATcov_RTS.Buffers.Lists.${ada_lib_name};
+with GNATcov_RTS.Traces.Output;
+% endif
+
 package body ${ada_lib_name}.Init is
 
    procedure Initialize is null;
 
+   % if emitter.coverage:
```

Comment:
OK you can probably disregard my earlier comment about `Init` then ^^

Reply:
We still can get rid of this package. ;-) I’ll just do it as another step to avoid messing with the commit history.